content/200-orm/200-prisma-client/150-using-raw-sql/100-typedsql.mdx

```diff
@@ -211,4 +217,8 @@ const result = await prisma.$queryRawUnsafe(
 );
 ```
 
-In this example, the columns to be selected are defined dynamically and included in the SQL query. While this approach provides flexibility, it requires careful attention to security, particularly to avoid SQL injection vulnerabilities. Additionally, using raw SQL queries means foregoing the type-safety and DX of TypedSQL.
\ No newline at end of file
+In this example, the columns to be selected are defined dynamically and included in the SQL query. While this approach provides flexibility, it requires careful attention to security, particularly to [avoid SQL injection vulnerabilities](/orm/prisma-client/using-raw-sql/raw-queries#sql-injection-prevention). Additionally, using raw SQL queries means foregoing the type-safety and DX of TypedSQL.
+
+## Acknowledgements
+
+This feature was heavily inspired by [PgTyped](https://github.com/adelsz/pgtyped) and [SQLx](https://github.com/launchbadge/sqlx). Additionally, SQLite parsing is handled by SQLx.
```

Comment:
Looks like this is lined up wierd? Might just be mu window. Did we want to mention PgTyped? I didn't  see that one